src/components/Card/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { FC } from 'react';
+import classNames from '@/components/Card/card.module.scss';
+
+type Props = {
+  title: {
+    text: string,
+    link?: string
+  },
+  data: {
+    key: string,
+    value: string
+  }[],
+  participants?:
+  {
+    userName: string,
+    imgUrl: string,
+  }[],
+  button?: {
+    text: string,
+    link?: string,
+    onClick?: () => void
+  };
+};
+
+const Card: FC<Props> = ({
+  title, data, participants = undefined, button = undefined,
+}) => {
+  const { text: tileText } = title;
+
+  function informationElement(key: string, value: string) {
+    return (
+      <span className={classNames.statusElement}>
+        <span className={classNames.statusLable}>{`${key}: `}</span>
+        <strong>{value}</strong>
+      </span>
+    );
+  }
+
+  return (
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events
+    <div
+      className={classNames.card}
+      onClick={() => {
+        if (title.link !== undefined) {
+          window.open(title.link, '_blank');
+        }
+      }}
+      role="button"
+      tabIndex={0}
+    >
+
+      <span className={classNames.prTitle}>{tileText}</span>
+
+      <div>
+        {data.map((pair) => (
+          <div>
```

Comment:
key is missing here.
When we use `map` we should also provide the `key` attribute.
you can read more [here](https://reactjs.org/docs/lists-and-keys.html#keys)

Reply:
Added a key!

https://github.com/Real-Dev-Squad/website-status/commit/f84f9afa7df2679012784eb4f1b1558a2496fdb4